src/App.js

```diff
@@ -92,6 +108,9 @@ class App extends Component {
     });
   };  // End loadClient()
 
+  /** Convert an object (ie devProps object) to a... string? @techExpertisePlease not too familiar with backticks and template literals... is this a string being created? Or a template? Or something else?
```

Comment:
It's creating an array of strings - a list of class names. The strings used are the keys of the object.

Reply:
Also, @knod are you sure it's an array of strings and not just a single, space separated, concatenated string that it returns? (That was my reading...)

Reply:
Template literals: We're actually hoping to convert as much of the code base as we can to those - it'll mean less escaping of actual quotation marks of any kind. To me personally, it seems like if we only use them when there are quotes around, that inconsistency would be more confusing. Maybe I'm off base there, though. It can definitely be discussed.

Returned value: Sorry, you're right, it's a string of words separated by spaces.